cclient/request/attach_to_tangle.h

```diff
@@ -14,6 +14,8 @@ extern "C" {
 
 #include "types/types.h"
 
+#define ATTACH_TO_TANGLE_MIN_MWM 18
```

Comment:
mainnet is 14

Reply:
ah, 18 is in the past.
Docs need an update though. 
https://iota.readme.io/v1.5.0/reference#attachtotangle

Reply:
yeah :)